src/devices/MotorHat/DCMotor3Pwm.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Iot.Device.DCMotor;
+using System;
+using System.Device.Pwm;
+
+namespace MotorHat
+{
+    internal class DCMotor3Pwm : DCMotor
+    {
+        private PwmChannel pwmPin;
+        private PwmChannel in1Pin;
+        private PwmChannel in2Pin;
+        private double speed = 0;
+
+        public DCMotor3Pwm(PwmChannel pwm, PwmChannel in1, PwmChannel in2) : base(null)
+        {
+            this.pwmPin = pwm;
+            this.pwmPin.DutyCycle = speed;
```

Comment:
pwm used as regular pin? 😄

Reply:
yup, that's how they made it, works fine though, but I doubt any would do this in other scenarios..